Front-end-Projects/Intermediate/Spotify-Clone/README.md

```diff
@@ -0,0 +1,69 @@
+<h1 align='center'><b>💥 Spotify Clone💥</b></h1>
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h3 align='center'>Tech Stack Used 🎮</h3>
+
+
+<div align='center'>
+
+  ![HTML5](https://img.shields.io/badge/html5-%23E34F26.svg?style=for-the-badge&logo=html5&logoColor=white)
+  ![CSS3](https://img.shields.io/badge/css3-%231572B6.svg?style=for-the-badge&logo=css3&logoColor=white)
+
+</div>
+
+
+![Line](https://github.com/Avdhesh-Varshney/WebMasterLog/assets/114330097/4b78510f-a941-45f8-a9d5-80ed0705e847)
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: Description 📃
+
+<div>
+  <p>This Project is a simple HTML and CSS implementation of the Spotify UI Clone. This project aims to showcase the UI design of spotify. This project is powered by HTML and CSS</p>
+</div>
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: How to run it? 🕹️
+
+<div>
+  <p>To run this project locally, follow these steps:
+
+1. Fork the repository.
+
+2. Clone the repository to your local machine:
+    git clone https://github.com/your-username/Product-Landing-Page.git
+
+3. Open the project folder in your preferred code editor, now you can view website in live.
+
+</p>
+</div>
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: Screenshots 📸
+<!-- add the screenshot of the project (Mandatory) -->
+
+<img src="screenshot 1.webp">
```

Comment:
![image](https://github.com/Avdhesh-Varshney/WebMasterLog/assets/114330097/6621a784-aa44-44aa-ba60-2ac426bb493d)

Reply:
sorry for the confusion, can you kindly review my new changes in the PR.